src/Pages/Sender/EnterPrice.tsx

```diff
@@ -60,6 +60,8 @@ const EnterPrice: React.FC = () => {
               value={objecInfo.price !== 0 ? objecInfo.price : undefined}
               onChange={handleInputChange}
               placeholder={t('requestCost.pricePlaceholder') || ''}
+              suffix="CHF"
+              type="number"
```

Comment:
This doesn't fully do what I think it should: I can still enter letters (when I start with letters and then numbers - doesn't work the other way around: numbers and then letters wipes the field), but then it simply goes away entirely when I go to the next page:

![image](https://github.com/portato-io/portatoapp/assets/24309085/605f0a03-f66d-48e9-b825-07174b893f22)
[NEXT PAGE]
![image](https://github.com/portato-io/portatoapp/assets/24309085/86e0944f-80f7-4db4-b4e1-64df6d486644)

In fact; it doesn't allow me to enter a dot or comma - that wipes the number altogether as well.

Reply:
This one is strange, it's working on Chrome.
Just tried on Firefox and I have the same behavior as you
Are you using Firefox/safari?

Reply:
The new push should fix it ;)